civiform_config.example.sh

```diff
@@ -21,9 +21,13 @@
 # - Commit sha from https://github.com/civiform/cloud-deploy-infra if USE_DOCKER=false
 # - "latest" to use latest version of either docker image or code from the repo, 
 #    depending on USE_DOCKER flag.
-
 export CIVIFORM_CLOUD_DEPLOYMENT_VERSION="latest"
 
+# REQUIRED
+# Version of the CiviForm server to use.
+# Needs to be a version from https://github.com/civiform/civiform/releases.
+export CIVIFORM_VERSION="v1.8.2"
```

Comment:
I think now we'll need to regularly update this value to be the latest version. We can make it part of release tasks.

Reply:
What do you think about having "latest" be in the example config, then updating the [deploy docs](https://docs.civiform.us/it-manual/sre-playbook/terraform-deploy-system#steps-to-run) to change the version to a specific release?

Reply:
I am weary adding more toil to the release process.  I agree explicit version is better.  Having "latest" in the example config is good for the "default config just works and additional config is easy to change" approach.

Do you think having a step to update the value to an explicit version in the deploy docs and adding a warning in the CLI if the value is "latest" is enough warning and a fair compromise to not add the release toil?